admin-part/src/main/kotlin/Plugin.kt

```diff
@@ -625,8 +625,8 @@ class Plugin(
                         send(buildVersion, Routes.Build.MethodsCoveredByTest.Summary(test), summary)
                         send(buildVersion, Routes.Build.MethodsCoveredByTest.All(test), all)
                         send(buildVersion, Routes.Build.MethodsCoveredByTest.Modified(test), modified)
-                        send(buildVersion, Routes.Build.MethodsCoveredByTest.Unaffected(test), new)
-                        send(buildVersion, Routes.Build.MethodsCoveredByTest.New(test), unaffected)
+                        send(buildVersion, Routes.Build.MethodsCoveredByTest.New(test), new)
+                        send(buildVersion, Routes.Build.MethodsCoveredByTest.Unaffected(test), unaffected)
```

Comment:
how the order of sending topic influence on front-end? 
why will the "new" methods be displayed as they should?

Reply:
1. The order of sending does not affect at all, because we send this data to the admin, it saves them in the database, and only on request from the front we receive data
2.Did not quite understand the question.
As you can see from the changes, the data from for different topics were confused, new methods were marked as not unaffected, and unaffected as new

Reply:
ok, I see, I thought that you change the order of them. Approved )